src/Authoring/WinRT.SourceGenerator/DiagnosticRules.cs

```diff
@@ -45,16 +41,15 @@ private static DiagnosticDescriptor MakeRule(string id, string title, string mes
         public static DiagnosticDescriptor ParameterNamedValueRule = MakeRule(
             "WME1092",
             "Parameter Named Value Rule",
-            ("The method {0} has a parameter named {1} which is the same as the default return value name. "

```

Comment:
Open question on if we support the attribute, but we can keep this diagnostic for now.

Reply:
I think you meant this for DefaultOverload ? I noticed you have multiple overloads of same arity but no attribute, added the attribute since we are blocking those scenarios. 

The diagnostic you commented on is for the parameter name conflict (__retval)